src/main/kotlin/com/cognifide/gradle/aem/instance/LocalInstance.kt

```diff
@@ -75,5 +75,8 @@ class LocalInstance(
     @get:JsonIgnore
     override var packages: ListResponse? = null
 
+    @Transient
```

Comment:
this will be moved probably to separate caching service (one instance per build, not project)